### ISSUE TITLE
Different angular errors

### ISSUE THREAD

**Issue by saxari** [yuanming-hu/fc4#41]
Hi Yuanming,

I noticed that I get different results when I use the loss function in train and in test. Can you tell me please what is the difference between these two loss functions? (angular_error function gives me smaller result when I use the same input)
get_angular_loss and angular_error

thanks in advance,
Sahar
